wellregistry/registry/tests/test_admin.py

```diff
@@ -39,17 +64,96 @@ def test_invalid_when_site_name_blank(self):
+        self.form_data['dec_long_va'] = ''
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
+        self.form_data['dec_long_va'] = '     '
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
+    def test_invalid_when_horizontal_datum_blank(self):
+        self.form_data['horizontal_datum'] = ''
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
+    def test_invalid_when_altitude_blank(self):
+        self.form_data['alt_va'] = ''
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
+        self.form_data['alt_va'] = '     '
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
+    def test_invalid_when_altitude_units_blank(self):
+        self.form_data['altitude_units'] = ''
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
+    def test_invalid_when_altitude_datum_blank(self):
+        self.form_data['altitude_datum'] = ''
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
+    def test_invalid_when_national_aquifer_blank(self):
+        self.form_data['nat_aqfr'] = ''
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
+    def test_invalid_when_site_type_blank(self):
+        self.form_data['site_type'] = ''
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
     def test_valid_when_display_flag_true_sn_flags_false(self):
         self.form_data['display_flag'] = True
         form = MonitoringLocationAdminForm(self.form_data)
         self.assertTrue(form.is_valid())
 
+    def test_invalid_when_site_type_well_and_aquifer_type_blank(self):
+        self.form_data['site_type'] = 'WELL'
+        self.form_data['aqfr_type'] = ''
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
+    def test_invalid_depth_unit_blank(self):
+        self.form_data['well_depth'] = '5'
+        self.form_data['well_depth_units'] = ''
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
+    def test_invalid_depth_unit_not_blank(self):
+        self.form_data['well_depth'] = ''
+        self.form_data['well_depth_units'] = '1'
+        form = MonitoringLocationAdminForm(self.form_data)
+        self.assertFalse(form.is_valid())
+
     def test_valid_when_display_flag_true_and_wl_sn_flags_true(self):
         self.form_data['display_flag'] = True
         self.form_data['wl_sn_flag'] = True
         self.form_data['wl_well_purpose'] = 'Other'
         self.form_data['wl_well_type'] = 'Trend'
-        self.form_data['wl_baseline_flag'] = True
```

Comment:
The baseline flag validation was just that it had to be not null.  Now that it's a boolean type, that is always the case.  So, I removed the wl_baseline_flag parts from the test.